frontend/src/metabase/query_builder/actions/core/updateQuestion.unit.spec.ts

```diff
@@ -811,6 +814,69 @@ describe("QB Actions > updateQuestion", () => {
     });
   });
 
+  describe("table column sync", () => {
```

Comment:
I think we should be adding tests to `metabase-lib`, closer to the sync columns method
Maybe [here](https://github.com/metabase/metabase/blob/768ba0577fccc8c4ec9cc16addf05fbdf16af4d9/frontend/test/metabase/lib/dataset.unit.spec.js#L29) would be a better place?

Reply:
That... would make way more sense haha. Will move those